HookInit.hpp

```diff
@@ -66,7 +66,7 @@ static void ReplaceFunction(void** funcPtr)
 	VirtualProtect(funcPtr, sizeof(*funcPtr), PAGE_READWRITE, &dwProtect);
 	wrapped_function::origFunction = **reinterpret_cast<decltype(wrapped_function::origFunction)*>(funcPtr);
 
-	*funcPtr = wrapped_function::Hook;
+	*funcPtr = reinterpret_cast<void*>(wrapped_function::Hook);
```

Comment:
One more thing I forgot to mention before - won't `*funcPtr = &wrapped_function::Hook;` eliminate the need to typecast?

Reply:
It does seem to eliminate that